ramls/userTenant.json

```diff
@@ -18,6 +18,22 @@
     "tenantId": {
       "description": "Primary tenant of the user for single-sign-on",
       "type": "string"
+    },
+    "centralTenantId": {
+      "description": "Central tenant of the user",
```

Comment:
Better to rename: Central tenant id in the consortium

Reply:
Description updated